index.html

```diff
@@ -4975,6 +5005,21 @@ <h2 id="AudioContextInfo">
                 Represents the sample rate of the associated
                 <a><code>BaseAudioContext</code></a>.
               </dd>
+              <dt>
+                boolean connected
+              </dt>
+              <dd>
+                Indicates that the associated <a>AudioWorkletNode</a> is
+                connected to one or more inputs in the audio graph. This
+                attribute is only updated immediately prior to an invocation of
+                <code>process()</code>. By examining this attribute, an
+                <a>AudioWorkletProcessor</a> can determine its lifetime via the
+                return value from its <code>process()</code> method, in a way
+                that depends on its connection status. For example, many
+                built-in node types exhibit a <a href=
+                "#dfn-tail-time">tail-time</a>, and this attribute permits
+                <a>AudioWorkletNodes</a> to provide the same thing.
```

Comment:
After looking at this, empty buffers are problematic since the number of channels can vary depending on the inputs as well as on the channel count configuration and in this case there are no inputs. My redraft instead uses zero input channels to indicate the lack of inputs (rather than an indeterminate number of input channels containing zero samples each).